cmd/viam-agent-syscfg/main.go

```diff
@@ -66,15 +66,19 @@ func main() {
 	}
 
 	log.Debugf("Config: %+v", cfg)
+
+	// exact text "startup complete" is important, the parent process will watch for this line to indicate startup is successful
+	log.Info("agent-syscfg startup complete")
```

Comment:
Moved this above the one-shots, as the new upgrade control can take a long/unpredictable time to install when bandwidth is really slow, so don't want to block startup or fail healthchecks.

Reply:
what does 'one shot' mean? actions that are only time-consuming the first time they're run?

if the one shot actions fail, is that visible anywhere / alertable?

Reply:
Yes, it logs errors if it fails. They only run at startup OR config change. They don't run continuously like, say, provisioning does.